hazelcast/src/main/java/com/hazelcast/query/impl/BaseIndexStore.java

```diff
@@ -145,13 +145,6 @@ private Comparable sanitizeScalar(Object input) {
         }
     }
 
-    void markIndexStoreExpirableIfNecessary(QueryableEntry record) {
```

Comment:
I'm a bit lost here. Now we don't update the `isIndexStoreExpirable` field anywhere but we still use the return value from `BaseIndexStore#isExpirable` in three places. Can you elaborate on this change?

Reply:
leftover, removed that method and related stuff.

Reply:
Still, can you elaborate a bit on how it's safe to remove all of that code? With the removal, the dependant code now acts as if `isExpirable() == false`, always.

Reply:
All filtering for expiry is done in one place now. In `Indexes#query` method,`resultFilter` will check for expiry. See the related part in that method:
```
   if (result != null && resultFilter != null) {
            return IterableUtil.filter(result, resultFilter);
        } else {
            return result;
        }
```

Reply:
Thanks for the explanation!